index.src.html

```diff
@@ -519,14 +520,18 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   execute the following step:
 
   9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+      <a>`Clear-Site-Data`</a> and the request's <a>credentials mode</a> permits
+      modifying credentials, then execute [[#clear-response]] on |response|.
```

Comment:
(Note that this would be simpler if we moved the hook to [HTTP-Network fetch](https://fetch.spec.whatwg.org/#http-network-fetch), which would simplify the service worker bits as well.)